src/app/search/state/search.service.ts

```diff
@@ -55,6 +55,15 @@ export class SearchService {
     });
   }
 
+  setPageSize(pageSize: number){
+    this.searchStore.setState(state => {
+      return {
+        ...state,
+        pageSize:pageSize
```

Comment:
There is typically a space after the colon, not sure if it is enforced in codestyle.